openapi/components/schemas/QuoteOrderItem.yaml

```
@@ -0,0 +1,108 @@
+type: object
+required:
+  - plan
+properties:
+  quoteItemId:
+    description: ID of the quote item.
+    readOnly: true
+    type: string
+    example: qti_0YV7DES3WPC5J8JD8QTVNZBZNZ
```

Please fix the incorrect prefix `qti` to `qt_itm`.

```suggestion
    example: qt_itm_01HXCEQNR3F1G2A6RX6HPS3KFY
```